torch/csrc/jit/autodiff.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
nit: autodiff traditionally means tape-and-recording stuff. So we should probably call this `symbolic_diff.h`